test/common/testlib.py

```diff
@@ -55,7 +55,7 @@
     'arg_parser',
     'Browser',
     'MachineCase',
-    'PersistentMachineCase',
+    'nonDestructive',
```

Comment:
ugh.  camelCase!  let's avoid the arguments by observing that nondestructive is one word :rofl: